assets/src/edit-story/components/panels/design/link/link.js

```diff
@@ -210,6 +210,7 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
         }
         isIndeterminate={isMultipleUrl}
         aria-label={__('Element link', 'web-stories')}
+        hasError={displayLinkGuidelines}
```

Comment:
Fixing this since every time when adding a shape, a console warning is yelling about `hasError` missing.